doc/30-namingscheme.md

```diff
@@ -0,0 +1,80 @@
+# Naming scheme
+This document will provide basic informations the naming scheme used within this project. It is essential to read, before contributing.
+
+### Basics
+The basics of the naming scheme are pretty easy and should be comprehended after taking a first glance at one of the config files.
+
+The **id** should be like this *icinga_NAMEOFLOGMESSAGE*, aswell as **add_tag**.
+The **add_field** section with **[icinga][eventtype]** always equals the name provided in the **id**, with _ to help with the readability.
+Every **grok filter** will get the following **tags_on_failure** _grokparsefailure and icinga_NAMEOFLOGMESSAGE_failed.
+```sh
+Example:
```

Comment:
This line should not be part of the code block

Reply:
"Example:" is outside the code block now, my bad.